cpp/cppsdk/gsdk.h

```diff
@@ -7,6 +7,7 @@
 #include <unordered_map>
 #include <functional>
 #include <exception>
+#include <stdexcept>
```

Comment:
I don't see this as required in gsdk.h.
I also don't see where this might be required in gsdk.cpp, but if it is (please verify), please move it to the cpp file, instead of the .h file.